poetry.lock

```diff
@@ -59,14 +59,14 @@ pytz = ">=2015.7"
 
 [[package]]
 name = "certifi"
-version = "2021.10.8"
+version = "2022.12.7"
```

Comment:
Does it need to be bumped again, now that 2022.12.7 has passed?  Sorry if this is a silly question.

Reply:
Certifi library is using dates as their version tag. so it doesn't matter if the date passed, but it means that the `2022.12.7` is a newer version than the one from `2021.10.8` and probably should be updated to it in some time.